From: https://github.com/void-linux/void-packages/tree/7bfdb7ab051fbb43309c764ed4613d39ee590679/srcpkgs/lshw/patches/

Fix GCC warning "array subscript is above array bounds [-Warray-bounds]"

--- a/src/core/dmi.cc
+++ b/src/core/dmi.cc
@@ -511,7 +511,7 @@
   if (num <= 0x0A)
     return _(memory_array_location[num]);
   if (num >= 0xA0 && num < 0xA4)
-    return _(jp_memory_array_location[num]);
+    return _(jp_memory_array_location[num - 0xA0]);
   return "";
 }
 
